seissolxmdf/README.md

```diff
@@ -0,0 +1,22 @@
+seissolxdmf
+===============
+A python reader for SeisSol xdmf output (posix or hdf5) and hdf5 meshes.
+Below is an simple example, illustrating the use of the module:
+```python
+import seissolxdmf as sx
+fn = 'test-fault.xdmf'
+# Number of cells
+nElements = sx.ReadNElements(fn)
+# Read time step
+dt = sx.ReadTimeStep(fn)
+# Read number of time steps
+ndt = sx.ReadNElements(fn)
+# load geometry array as a numpy array of shape ((nodes, 3))
+geom = sx.ReadGeometry(fn)
+# load connect array as a numpy array of shape ((nElements, 3 or 4))
```

Comment:
Can you explain what the entries of the connect array are?

Reply:
ok